modules/potree/test/potree-source.spec.ts

```diff
@@ -1,11 +1,13 @@
 import test from 'tape-promise/tape';
 import {PotreeSource} from '@loaders.gl/potree';
 
-const POTREE_HIERARCHY_CHUNK_URL = '@loaders.gl/potree/test/data/lion_takanawa';
+const POTREE_BIN_URL = '@loaders.gl/potree/test/data/lion_takanawa';
+const POTREE_LAZ_URL =
```

Comment:
Maybe use version numbers here? POTREE_V17_URL, POTREE_V18_URL etc?

Reply:
The criteria for these datasets here is content type (BIN/LAZ). 1.7 and 1.8 are almost the same. The significant difference that we don't support bin content a the moment.